### PR TITLE
Quickbooks Connector 1.0 doc

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

**Review by laszloczol — DISMISSED** [review]

File: quickbooks/1.0/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+* The QuickBooks target resource
+* Anypoint Platform
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases For the Connector
+
+Common use cases for QuickBooks Online Connector include the following:
+
+* Eliminating data staleness and manual entry errors by automatcally syncing of your ERP data in QuickBooks Online with your accounts and customer data in Salesforce.
```
Comment (mcherem) [inline quickbooks/1.0/modules/ROOT/pages/index.adoc]: automatcally > automatically

File: quickbooks/1.0/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+* Reducing cycle times by automating business critical flows across QuickBooks Online and other cloud-based and on-premises systems.
```
Comment (mcherem) [inline quickbooks/1.0/modules/ROOT/pages/index.adoc]: business critical > business-critical

File: quickbooks/1.0/modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (mcherem) [reply quickbooks/1.0/modules/ROOT/pages/index.adoc]: From this line:
syncing of your ERP data 

Is the 'of' necessary?

**Review by mcherem — DISMISSED** [review]
Peer review.

**Review by laszloczol — APPROVED** [review]

**Review by laszloczol — APPROVED** [review]

**Review by mcherem — APPROVED** [review]
re approve

**Pull request merged**